src/basics/Text.js

```diff
@@ -222,7 +222,7 @@ export const Preformatted = styled.pre`
     line-height: inherit;
     white-space: pre-wrap;
     max-width: 90rem;
-    width: 1000rem;
+    width: 100%;
```

Comment:
We need to force this to something super wide in order to avoid wrapping URLs. I noticed that it's always at 90rem though, so might be be better to change to `width: 90rem;`

Reply:
Removed it because the `<Text />` that docs was using was a placeholder